.github/workflows/make_release.yml

```
@@ -0,0 +1,17 @@
+name: Make a Github release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        with:
+          prerelease: true
```

something like this:
```suggestion
          prerelease: ${{ contains(github.ref_name, "-rc") }}
```
should work. "prerelease" will be true if the tag contains "-rc", and false if it doesn't.